### PR TITLE
Feature/green border

### PR DESCRIPTION
**Pull request by anasanjuan**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -8,7 +8,7 @@
     <style>
         body {
             background-color: orange;
-            border: 10px solid skyblue;
+            border: 10px solid greenyellow;
```
Comment (fedeCucchiararo) [inline index.html:11]: hey!!! Why that color?

File: index.html [hunk 2]
```diff
@@ -8,7 +8,7 @@
     <style>
```
Comment (fedeCucchiararo) [inline index.html:8]: hey! move styles to files! IMMEDIATELY

Comment (anasanjuan) [reply index.html:8]: true... but you could have done it before! MOVE YOUR FINGERS!!

Comment (fedeCucchiararo) [reply index.html:8]: Sure. I'll do it after the break!

**Pull request merged**